mmcls/apis/list_registry.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+
+def list_registry(specific_registry=None):
+    """List the registry of MMClassification.
+
+    When specific_registry is None, list all available filed of registry
+
+    When specific_registry is given, will list all modules in
+        `getattr(registry, specific_registry)`
+
+    Args:
+        specific_registry (str|None): which filed of registry need to be listed
+    """
+    import pprint
+
+    from mmengine.registry import Registry
+
+    from mmcls import registry
+    from mmcls.utils import register_all_modules
+
+    register_all_modules()
```

Comment:
Better to use `register_all_modules(init_default_scope=False)`, which won't influence the default scope settings.